check_manifest.py

```diff
@@ -377,11 +378,24 @@ def get_vcs_files():
 
 
 def normalize_names(names):
+    """Normalize file names.
+    """
```

Comment:
[PEP-257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) says "The closing quotes are on the same line as the opening quotes" for one-line docstrings.